src/components/modes/student/StudentView.js

```diff
@@ -32,27 +32,32 @@ const useStyles = makeStyles(theme => ({
 export const StudentView = props => {
   const { t, initialTimeValue, direction, tool } = props;
   const classes = useStyles();
+  const MillSecsToMin = 60;
```

Comment:
This is hours in min, not millis. Double check that it's right and use the right name in all caps. `ONE_HOUR_IN_MINS`.